File: src/sbt-test/lagom-bundle-plugin/multi-services-one-project-same-service-name-scala/payment-impl/src/main/scala/impl/DebitServiceImpl.scala
package impl

import akka.NotUsed
import api.DebitService
import scala.concurrent.Future
import com.lightbend.lagom.scaladsl.api.ServiceCall

class DebitServiceImpl extends DebitService {
  override def debit = ServiceCall { x =>
    Future.successful(NotUsed)
  }
}

File: src/sbt-test/lagom-bundle-plugin/multi-services-one-project-same-service-name-java/payment-impl/src/main/java/impl/DebitServiceImpl.java
package impl;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import java.util.concurrent.CompletableFuture;
import javax.inject.Inject;
import api.DebitService;

import akka.stream.javadsl.Source;

public class DebitServiceImpl implements DebitService {

  @Override
  public ServiceCall<NotUsed, NotUsed> debit() {
    return request -> CompletableFuture.completedFuture(NotUsed.getInstance());
  }
}
